advanced/index.md

```diff
@@ -49,8 +49,8 @@ Explore the structure and way change is preserved in Git.
 ---
 
 {% capture lab %}
-1. Understand the way Git preserves history and data objects
-2. Utilize shorthand for navigation commit history
+1. Understand the way Git preserves data
```

Comment:
My suggestion as a happy medium would be:

> 1. Understand the way Git preserves history and data

This removes the need to concern readers with what "objects" is from a objectives list standpoint.